net/miniupnpd/Makefile

```diff
@@ -9,7 +9,7 @@ include $(TOPDIR)/rules.mk
 
 PKG_NAME:=miniupnpd
 PKG_VERSION:=2.0.20180503
-PKG_RELEASE:=1
+PKG_RELEASE:=3
```

Comment:
Umm.... what happened to `2`?

Reply:
2 had been used in first commit

Reply:
There were 2 commits in the 1 PR, with makefile version bumps in both, to be applied in sequence.  The idea being to be able to revert the individual changes if needed.